build.gradle

```diff
@@ -1,7 +1,6 @@
 subprojects {
     ext.version_number   = "0.1.2"
     ext.group_info       = "org.signal"
-    ext.protocol_version = "2.8.1"
```

Comment:
Do we not want to track this?

Reply:
That's Signal's versioning scheme, not ours. Once we stabilize we'll likely start releasing versions too, but for now we're just pinning to commit hashes via Jitpack.